pkg/sum/sum.go

```diff
@@ -123,27 +137,36 @@ type sum struct {
 	biosCfg               BiosCfg
 	machineType           machineType
 	uefiNetworkBootOption string
+	secureBootEnabled     bool
 }
 
 func New() (Sum, error) {
 	return &sum{}, nil
 }
 
-// EnsureUEFIBoot updates BIOS to UEFI boot.
-func (s *sum) EnsureUEFIBoot(reboot bool) error {
+// UpdateBIOS updates BIOS to UEFI boot and disables CSM-module if required.
+// If returns whether machine needs to be rebooted or not.
+func (s *sum) UpdateBIOS() (bool, error) {
```

Comment:
wouldn´t be ConfigureBios a better name ? UpdateBios could mean we update the version of the bios which is not the case.

Reply:
Done. I have also tested this in our testing environment.